tomcat-logs/pom.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>pentaho-platform-ce-parent</artifactId>
+        <groupId>pentaho</groupId>
+        <version>9.4.0.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>pentaho-tomcat-logs</artifactId>
+    <version>9.4.0.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
```

Comment:
This is building on JDK11 now, and should use the parameter already defined here: https://github.com/pentaho/maven-parent-poms/blob/master/pom.xml#L210

You shouldn't override this setting in lower projects unless it's essential; we're currently still targeting JDK8 for compatibility with certain big data drivers.